llzk-sys/build-support/src/llzk.rs

```diff
@@ -0,0 +1,78 @@
+use anyhow::{Context, Result};
+use bindgen::Builder;
+use cc::Build;
+use std::path::{Path, PathBuf};
+
+use super::config::{bindgen::BindgenConfig, cc::CCConfig, cmake::CMakeConfig};
+
+pub const LIBDIR: &'static str = "lib";
+
+pub struct LlzkBuild<'a> {
+    path: PathBuf,
+    src_path: &'a Path,
+}
+
+impl<'a> LlzkBuild<'a> {
+    fn new(src: &'a Path, dst: PathBuf) -> Self {
+        Self {
+            path: dst,
+            src_path: src,
+        }
+    }
+
+    pub fn path(&self) -> &Path {
+        &self.path
+    }
+
+    pub fn emit_cargo_instructions(&self) -> Result<()> {
```

Comment:
Are these emitted instructions meant to be used by the user, or are they more for informational purposes? If meant to be run by the user, it might be nice to emit to a file or something.